pkg/gecko_sdk/Kconfig

```diff
@@ -1,11 +1,29 @@
 # Copyright (c) 2021 HAW Hamburg
+#               2022 SSV Software Systems GmbH
 #
 # This file is subject to the terms and conditions of the GNU Lesser
 # General Public License v2.1. See the file LICENSE in the top level
 # directory for more details.
 #
 
-config PACKAGE_GECKO_SDK
+## Definition of Gecko SDK specific features
+config HAS_GECKO_SDK_LIBRAIL_NONFPU
+    bool
+    help
+        Indicates that the CPU offers librail support if the FPU is disabled.
+        Librail is shipped as pre-compiled blobs. Thus, we have to adapt to
+        their choice how to process floats.
+
+config HAS_GECKO_SDK_LIBRAIL_FPU
+    bool
+    help
+        Indicates that the CPU offers librail support if the FPU is enabled.
+        Librail is shipped as pre-compiled blobs. Thus, we have to adapt to
+        their choice how to process floats.
```

Comment:
So RAIL is the radio HAL, right? They use floats in their radio driver?!?

![wtf](https://media1.giphy.com/media/doJrCO8kCAgNy/giphy.webp?cid=dda24d50dtjxfvob0ufare7ozixr6he26jhp3d70d7j16vp9&rid=giphy.webp&ct=g)

Reply:
I've seen many surprising design choices made by Silabs ... but yes, I wouldn't expect a radio HAL using floats, as well :D But I can't tell - that's the problem with those vendor-specific blobs. Actually, I spent way too long with the integration because of those black boxes. And they weren't willing to let me look inside even after dozes of exchanged emails :/